extension/js/roblox/games.js

```diff
@@ -353,6 +288,32 @@ switch (Extension.Singleton.executionContextType) {
 			types: ["xmlhttprequest"]
 		}, ["blocking", "requestHeaders", "extraHeaders"]);
 
+		chrome.webRequest.onBeforeRequest.addListener(request => {
+			const url = new URL(request.url);
+			if (!url.pathname.match(/^\/v1\/games\/\d+\/servers\/Public/i)) {
```

Comment:
Probably could have used a $ here.

```suggestion
			if (!url.pathname.match(/^\/v1\/games\/\d+\/servers\/Public$/i)) {
```